### ISSUE TITLE
OpenBSD md5 port

### ISSUE THREAD

**Issue by grylem** [janstary/ports#2]
maybe you can make also a port of OpenBSD md5?
Thanks.

**Comment by janstary** [comment]
Newer versions of MacOS do have `/sbin/md5`.

**Comment by grylem** [comment]
Yes but it's different macOS version from freeBSD so does not have some options like to work with a checklist.
